library/libraries.js

```diff
@@ -437,6 +437,20 @@ var d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests = {
         }
     },
 
+    '$mol_view': {
+        icon: 'mol',
+        url: 'http://mol.js.org/',
+        test: function(win) {
+            if (win.$mol_view) {
```

Comment:
```suggestion
            if (win.$mol_view || win.document.querySelector('$mol_view_root')) {
```
Would be preferable to having two, single-condition `if` blocks here.

Reply:
I think 2 simplest conditions is easier to understand than 1 complex condition.